index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Camilo Vela Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100&family=Roboto:wght@100&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <header class="header">
+      <a class="logo">John Doe</a>
+      <img class= "normal_button" src="images/Enabled.png" alt="normal_button">
+    </header>

```

Comment:
- [ ] please make sure to adjust the height of your header in order to match the one in Figma design

Reply:
Do I have to add mobile phone-related icons (time, phone signal, wifi signal, battery charge level)? I adjusted the height to match Figma Template, but if I add these icons, it would show up repeated when the webpage was opened on a mobile phone.